File: 15_09_2022/1684._Count_the_Number_of_Consistent_Strings.cpp
class Solution {
public:
    int countConsistentStrings(string al, vector<string>& wrd) {
        char arr[26]={0};
        int a=0;
        for(auto &k:al)
        {
            arr[k-97]=1;
        }
        int z=1;
        for(auto &x:wrd)
        {
            z=1;
            for(auto &xx:x)
            {
                if(arr[xx-97]==0)
                {
                    z=0;
                }
            }
            
            if(z)
            {
                a++;
            }
            
        }
        return a;
    }
};

File: 15_09_2022/1684._Count_the_Number_of_Consistent_Strings.py
class Solution:
    def countConsistentStrings(self, alw: str, wrd: List[str]) -> int:
        d=[0]*26
        ans=0
        for i in alw:
            t=ord(i)-97
            d[t]=1
        
        for i in wrd:
            xx=1
            for j in i:
                j=ord(j)-97
                if(d[j]==0):
                    xx=0
                    break
            if(xx):
                ans+=1
                
        return ans